src/api/app/assets/stylesheets/webui/application/image_templates.sass

```diff
@@ -3,17 +3,12 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 .break-words
-  /* These are technically the same, but use both */
+  // The three does the same, to make it work in more browser versions
```

Comment:
Aehm - *those* is the plural of that, which implies distance (to say it with merriam-webster: 'the one farther away or less immediately under observation or discussion'). As the css statements referenced to are directly behind the comment, I think you want the plural of this (**these**): 'what is stated in the following phrase, clause, or discourse'.

And as we're talking about grammar: the comma is an incorrect separator within the statement. 'These three do the same' is a sentense and the part after the comma creates what the grammatians call a comma splice or run-on sentense. Just that the part behind is not a sentense. A more appropriate separator would be a colon - or my personal preference - a dash.

But in this very example I suggest not to use a comma at all and state:
```
// These three make it work in different browsers
```

Sorry for my outburst :smiling_imp: